content/3-lifecycle/2-on-unmount/angular/time.component.ts

```diff
@@ -9,12 +9,12 @@ export class TimeComponent implements OnDestroy {
 	timer: number;
 
 	constructor() {
-		this.timer = window.setInterval(() => {
+		this.timer = setInterval(() => {
 			this.time = new Date().toLocaleTimeString();
 		}, 1000);
 	}
 
-	ngOnDestroy(): void {
-		window.clearInterval(this.timer);
+	ngOnDestroy() {
+		clearInterval(this.timer);
```

Comment:
suggestion/nit (non-blocking): I think the use of `setInterval` here is to make it comparable to the other technologies but would it make more sense to use `interval()`?